src/inpage-content.js

```diff
@@ -0,0 +1,105 @@
+
+const url = window.location.href;
+
+const action = parseQuery(url).action;
```

Comment:
suggestion (blocking): use the built-in `URL.searchParams` property so we can remove the `parseQuery` function & overhead of finding the `action` param.
```suggestion
const action = new URL(url).searchParams.get("action");
```

Reply:
didn't know this existed!! Awesome, thanks!